### PR TITLE
Update frankencoin

### PR DESCRIPTION
**Pull request by samclassix**

### CODE REVIEW

**Comment by llamabutler** [comment]
The adapter at projects/frankencoin exports TVL: 
        
 
 
    ethereum                  31.05 M
    
    total                    31.05 M

**Comment by samclassix** [comment]
> The adapter at projects/frankencoin exports TVL:
> 
> ```
> ethereum                  31.05 M
> 
> total                    31.05 M 
> ```

Yes. :)

File: projects/frankencoin/index.js [hunk 1]
```diff
@@ -1,14 +1,21 @@
...
 
 async function tvl(api) {
-  const tokensAndOwners = Collaterals.map(i => [i, MintingHub])
-  const { positions } = await cachedGraphQuery('frankencoin', 'https://api.thegraph.com/subgraphs/name/frankencoin-zchf/frankencoin-subgraph', '{ positions { position collateral } }')
-  positions.forEach(i => tokensAndOwners.push([i.collateral, i.position]))
-  return sumTokens2({ api, tokensAndOwners, })
+  const tokensAndOwners = [XCHFBridge];
+
+  // @dev: query of positions from minting hubs via frankencoin graph (ponder)
+  const { positionV1s } = await graphQuery('https://ponder.frankencoin.com/', '{ positionV1s { items { position collateral } } }');
```
Comment (waynebruce0x) [inline projects/frankencoin/index.js:11]: can we use cachedGraphQuery() here instead?

**Comment by llamabutler** [comment]
The adapter at projects/frankencoin exports TVL: 
        
 
 
    ethereum                  30.44 M
    
    total                    30.44 M

**Comment by llamabutler** [comment]
The adapter at projects/frankencoin exports TVL: 
        
 
 
    ethereum                  30.44 M
    
    total                    30.44 M

File: projects/frankencoin/index.js (hunk 1 above)
Comment (samclassix) [reply projects/frankencoin/index.js]: sure. adjusted those changes.

**Pull request merged**